tests/platform_tests/test_reboot.py

```diff
@@ -265,6 +265,8 @@ def test_watchdog_reboot(duthosts, enum_rand_one_per_hwsku_hostname, localhost,
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
+    if duthost.is_supervisor_node and duthost.facts['platform'] in ['x86_64-8800_rp_o-r0']:
```

Comment:
@vperumal 
Instead of adding a skip here for a specific HW SKU, can this be changed in the Cisco platform code on the handling of "sudo watchdogutil status" to error out or produce no output so that the test is naturally skipped with existing code?
When Cisco is ready to handle it, then you can change your code to respond with proper response and we do not need to change this test code again?
Have you considered this approach instead?

Reply:
@vperumal any comment ?